drivers/SPI.h

```diff
@@ -143,6 +143,14 @@ class SPI : private NonCopyable<SPI> {
      */
     virtual void unlock(void);
 
+    /** SPI block read dummy data
+      * SPI requires master to send dummy data, in order to perform read operation.
+      * Dummy bytes can be different for devices. Example SD Card require 0xFF.
+      *
+      * @param dummy    Dummy character to be transmitted while read operation
```

Comment:
Edits in bold:
Dummy ~character~ **byte** to be transmitted ~while~ **during** **a** read operation

This should probably be 
```C
* @param data
```